packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -764,31 +862,35 @@ const QueryBuilderGridResult = observer(
         >
           <DataGrid
```

Comment:
could we something similar here and separate the component 
something like `isAdvancedMode ? <datagrid options={advanced} : <datagrid options={normal}`.